tests/Routing/Adapter/LaravelTest.php

```diff
@@ -6,7 +6,6 @@
 use Illuminate\Events\Dispatcher;
 use Illuminate\Container\Container;
 use Dingo\Api\Routing\Adapter\Laravel;
-use Illuminate\Routing\RouteCollection;
```

Comment:
Already removed in https://github.com/dingo/api/commit/cc21964e0fb3c2eaa50954dfb27243c7caf4c1c7